rfcs/text/0008-sip-fields.md

```diff
@@ -0,0 +1,242 @@
+# 0000: SIP Fields
+<!-- Leave this ID at 0000. The ECS team will assign a unique, contiguous RFC number upon merging the initial stage of this RFC. -->
+
+- Stage: **0 (strawperson)** <!-- Update to reflect target stage. See https://elastic.github.io/ecs/stages.html -->
```

Comment:
Between the discussions we've had already and the tremendous work that @dainperkins has already invested, I suggest targeting stage 1 here.

What do others think?

Reply:
Agreed with stage 1 from the start